dockerfiles/bash/global.cylc

```diff
@@ -0,0 +1,3 @@
+[platforms]
+    [[_local_background_indep_tcp]]
+        hosts = localhost
```

Comment:
Just a quick reminder if this pr breaks (in case my PR (#3796) is merged first) -  these kinds of setups will require `install target = localhost` added.

Reply:
Thanks.

Reply:
Done.